src/Nancy.Bootstrappers.Unity/UnityNancyBootstrapper.cs

```diff
@@ -81,7 +81,11 @@ protected override INancyEngine GetEngineInternal()
         /// <returns>Container instance</returns>
         protected override IUnityContainer GetApplicationContainer()
         {
-            return new UnityContainer();
+            var container = new UnityContainer();
+
+            container.AddNewExtension<EnumerableExtension>();

```

Comment:
Yeah, I'm afraid you have to add the extension manually. And I'm not sure we want to add a custom extensibility point here. If you create the container - you should add extensions IMO. What if someone passed in a container with a similar extension?